src/presolve.jl

```diff
@@ -145,6 +148,9 @@ mutable struct PresolveData{T}
             ps.us[j] = (lv == T(-Inf)) ? zero(T) : T(Inf)
         end
 
+        # Variable types
+        ps.var_types = pb.var_types
```

Comment:
```suggestion
        ps.var_types = copy(pb.var_types)
```
so we can modify this without changing the original data.